client/src/pages/Login/Login.tsx

```diff
@@ -95,13 +96,17 @@ const Login = () => {
           </ul>
 
           <FormButton
+            size="large"
             type="submit"
             disabled={isSubmitting}
             className={styles.submitBtn}
           >
             Submit
           </FormButton>
         </Form>
+        <Link to="/signup" className={styles.signupBtn}>
```

Comment:
i would maybe just put it as a link and maybe in black or red and not with that background - but not critical